corehq/ex-submodules/casexml/apps/phone/management/commands/invalidate_sync_heads.py

```diff
@@ -0,0 +1,33 @@
+from django.core.management import BaseCommand, CommandError
+
+from casexml.apps.phone.models import SimplifiedSyncLog
+from casexml.apps.phone.dbaccessors.sync_logs_by_user import synclog_view
+
+
+class Command(BaseCommand):
+    """
+    Forces a 412 for a given user by creating bad state in the all synclogs
+    for the given user after the given date
+    """
+    args = "user_id date"
+
+    def handle(self, *args, **options):
+        if len(args) != 2:
+            raise CommandError("Usage is ./manage.py invalidate_sync_heads %s" % self.args)
+        user_id = args[0]
+        date = args[1]
```

Comment:
what format is the date in here? I think this could use some validation, as I'd be wary of accidentally blowing away information on synclongs that we don't need to.